addons/sourcemod/scripting/RepeatKillDetector.sp

```diff
@@ -98,7 +114,7 @@ public Action Command_ForceRepeator(int client, int argc)
 public Action Event_PlayerDeath(Handle event, char[] name, bool dontBroadcast)
 {
 	if(g_bBlockRespawn)
-		return Plugin_Continue;
+		return Plugin_Handled;
```

Comment:
keep it Plugin_Continue, it may break some things if we change this hook to EventHookMode_Pre